apps/token-manager/contracts/TokenManager.sol

```diff
@@ -179,8 +179,8 @@ contract TokenManager is ITokenController, IForwarder, AragonApp {
     */
     function revokeVesting(address _holder, uint256 _vestingId)
         external
-        authP(REVOKE_VESTINGS_ROLE, arr(_holder))
         vestingExists(_holder, _vestingId)
+        authP(REVOKE_VESTINGS_ROLE, arr(_holder))
```

Comment:
I don't fully get why the state check has to take precedence over the authentication one, specially in a case like this one where they are more disjoint, but I don't see any strong reason for the opposite either, so I'm fine with these changes.